mlir/test/Transforms/buffer-results-to-out-params-add-result-attr.mlir

```diff
@@ -0,0 +1,18 @@
+// RUN: mlir-opt -p 'builtin.module(buffer-results-to-out-params{add-result-attr})' -split-input-file -verify-diagnostics %s | FileCheck %s
+
+// CHECK-LABEL: basic
+// CHECK-SAME:  memref<f32> {bufferize.result})
```

Comment:
nit: I'd write this as `CHECK-LABEL: @basic({{.*}} : memref<f32> {bufferize.result})` for better readability.